site/js/components/hero-animated.js

```diff
@@ -53,25 +65,56 @@ export class HeroAnimated {
 
     /** @type {AnimationConfig} */
     this.animationConfig_ = {
-      container: this.elem_.querySelector(this.constants_.wrapperSelector),
+      container: this.elem_.querySelector(this.constants_.selectors.wrapper),
       animType: 'svg',
       loop: true,
       animationData: animationData,
       autoplay: false,
-      name: this.constants_.animationName,
+      name: this.constants_.animation.name,
       rendererSettings: {
         progressiveLoad: true,
-        className: this.constants_.animationItemClass,
       },
     };
 
     this.init_();
   }
 
   /**
-   * Inits the animation and sets the event handlers.
+   * Check for user preferences and use the theme to apply the corresponding animation.
+   */
+  async useTheme() {
+    if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
+      const phosphor = document.body.dataset.theme === 'phosphor';
+      let lottie;
+      let animationData;
+
+      if (phosphor) {
+        [{ default: lottie }, { animationData }] = await Promise.all([import('lottie-web/build/player/lottie_svg.min.js'), import('../animations/phosphor')]);
+      } else {
+        [{ default: lottie }, { animationData }] = await Promise.all([import('lottie-web/build/player/lottie_svg.min.js'), import('../animations/home')]);
+      }
+
+      this.loadAnimation(lottie, animationData);
+    } else {
+      this.showMotionFallback();
+    }
+  }
+
+  /**
+   * Removes the class that hides the fallback asset and adds it to the background Image.
+   */
+  showMotionFallback() {
+    this.fallbackImage_.classList.remove(this.constants_.hideClass);
+    this.backgroundImage_.classList.add(this.constants_.hideClass);
```

Comment:
Doesn't `phosphor` also need to be hidden for motion fallback?

Reply:
The phosphor asset is hidden by default when it's not a descendant of an element with `data-theme="phosphor"` within `_hero-animated.scss`.